### DISCUSSION TITLE
Problem changing ZigBee channel from 11 to 20

### DISCUSSION THREAD

**Discussion by DeekB1983** [Koenkk/zigbee2mqtt#10957] [General]
I have set up zigbee2mqtt on a raspberry pi 4 to run as a separate Zigbee2MQTT server as my home assistant is virtualised on hyper V.

I am install on Raspbian buster lite image and the O\S has been fully updated, I can get zigbee 2mqtt working and speaking to home assistant and the devices appear like they should.

My current config
frontend:
  port: 8080
  host: 192.168.25.41
homeassistant: true
permit_join: true
mqtt:
  base_topic: zigbee2mqtt
  server: 'mqtt://192.168.50.11'
  user: ************
  password: ***************
serial:
  port: /dev/ttyUSB0
  network_key:  *Generated key exists here*

This in general works but I am looking to change my channel to 20 or 25 too avoid WiFi overlap as I have some unify access points using channels 1 & 6

If I connect to the front end and change the channel from 11 to 20 I get prompted to restart, after a restart I am unable to access the qigbee2mqtt webpage, I can see the extra following config is added:
ikea_ota_use_test_url: false
  log_syslog:
    app_name: Zigbee2MQTT
    eol: /n
    host: localhost
    localhost: localhost
    path: /dev/log
    pid: process.pid
    port: 123
    protocol: tcp4
    type: '5424'
  channel: 20
device_options:
  legacy: false

Checking the log shows the following:
info  2022-01-22 17:20:42: Logging to console and directory: '/opt/zigbee2mqtt/data/log/2022-01-22.17-20-40' filename: log.txt
info  2022-01-22 17:20:42: Starting Zigbee2MQTT version 1.22.2 (commit #1a0a9a6e)
info  2022-01-22 17:20:42: Starting zigbee-herdsman (0.13.188)
error 2022-01-22 17:20:43: Configuration is not consistent with adapter state/backup!

If I remove the config above I am able to restart zigbee2mqtt but then I am back to being stuck on channel 11.

Am I doing something wrong or is there another way to change the channel - I am new to zigbee2mqtt so possibly I am missing soemthing I have read a lot of the website but not been able to overcome this issue yet

**Comment by DeekB1983** [comment]
I managed to fix this by setting the config in the configuration.yaml file and deleting the coordinator_backup.json file.

**Comment by almirus** [comment]
same
![121323](https://user-images.githubusercontent.com/16898685/202846971-d055d5b1-0539-47ef-8ffd-316e59d3c372.gif)

can't set another value

**Comment by Drealine** [comment]
Channel change can only permit while join is disable

**Comment by almirus** [comment]
When need to disable this input

**Comment by Drealine** [comment]
This is the button allow permit join should be disable to change channel
Ive the same problem with you and this is solve the problem. It prevent change channel when pairing
